src/screens/Ticketing/Purchase/index.tsx

```diff
@@ -1,28 +1,47 @@
 import React from 'react';
-import TravellersScreen from './Travellers';
+import TravellersScreen, {TravellersProps} from './Travellers';
 import {CreditCard as CreditCardScreen, Vipps as VippsScreen} from './Payment';
 import createDismissableStackNavigator from '../../../navigation/createDismissableStackNavigator';
 import {ActiveTicketsScreenName} from '../Tickets';
-import {ReserveOffer} from '../../../api/fareContracts';
+import {FareContractType, ReserveOffer} from '../../../api/fareContracts';
+import {DetailScreenRouteProp} from '../../TripDetailsModal/Details';
+import {RouteProp} from '@react-navigation/core';
 
-type PaymentParams = {offers: ReserveOffer[]};
+type TravellersParams = {
+  refreshOffer?: boolean;
+  fareContractType: FareContractType;
```

Comment:
Added the `fareContractType` as a param in navigation. Thought of setting it in the `TicketContext` instead, making it not necessary to pass it around between navigation stack screens. What is preferred?